code/src/components/HappyHeart.css

```diff
@@ -0,0 +1,74 @@
+.inputContainer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    min-width: 260px;
+    max-width: 450px;
+    padding: 0 5%;
+    margin: 20px 0;
+    border: 1px solid #000;
+    box-shadow: 6px 6px 0 #000;
+    word-break: break-word;
+}
+
+.heart_buttonWrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 2%;
+
+}
+.heartButton_Counter {
+    display: flex;
+    align-items: baseline;
+}
+
+.heartButton_Counter p {
+    color: gray;
+    font-size: small;
+}
+
+.heartButton {
+    width: 35px;
+    height: 35px;
+    border-radius: 50%;
+    cursor: pointer;
```

Comment:
The heart was very tricky to center! a tip that I did was to make margin-right: -5px on the side that was too much space. maybe not the best way but it helped!